lib/matcher.js

```diff
@@ -449,6 +507,9 @@ export const applyRule = (rule, input) => {
                 }
             }
         }))
+        output['outputPath'] = getPath(result, replacement)
+        output['outputAST'] = result
+        return result
```

Comment:
We'll eventually want to return `output` instead of `result`, but this will cause lots of tests to fail.  Can you keep that change separate so that this is easier to review?